RunOfflineAirPollutionAnalysisByVehicleCategoryTest fails in master branch

Hello,
after updating matsim-libs RunOfflineAirPollutionAnalysisByVehicleCategoryTest started to fail. The 5.5 release branch uses an old matsim version and for that reason does not fail.

https://github.com/matsim-scenarios/matsim-berlin/runs/4106057525?check_suite_focus=true#step:4:96163

```
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 11.199 sec <<< FAILURE! - in org.matsim.analysis.emissions.RunOfflineAirPollutionAnalysisByVehicleCategoryTest
test1(org.matsim.analysis.emissions.RunOfflineAirPollutionAnalysisByVehicleCategoryTest)  Time elapsed: 11.199 sec  <<< ERROR!
java.lang.RuntimeException: java.lang.RuntimeException: Was not able to lookup emissions factor. Maybe you wanted to look up detailed values and did not specify this in the config OR you should use another fallback setting when using detailed calculation OR values ar missing in your emissions table(s) either average or detailed OR... ? efkey: HEAVY_GOODS_VEHICLE; NOx; 13; 1; average; average; average
	at org.matsim.analysis.emissions.RunOfflineAirPollutionAnalysisByVehicleCategoryTest.test1(RunOfflineAirPollutionAnalysisByVehicleCategoryTest.java:44)
Caused by: java.lang.RuntimeException: Was not able to lookup emissions factor. Maybe you wanted to look up detailed values and did not specify this in the config OR you should use another fallback setting when using detailed calculation OR values ar missing in your emissions table(s) either average or detailed OR... ? efkey: HEAVY_GOODS_VEHICLE; NOx; 13; 1; average; average; average
	at org.matsim.analysis.emissions.RunOfflineAirPollutionAnalysisByVehicleCategoryTest.test1(RunOfflineAirPollutionAnalysisByVehicleCategoryTest.java:41)
```

--- a/src/test/java/org/matsim/analysis/emissions/RunOfflineAirPollutionAnalysisByVehicleCategoryTest.java
+++ b/src/test/java/org/matsim/analysis/emissions/RunOfflineAirPollutionAnalysisByVehicleCategoryTest.java
@@ -35,11 +35,11 @@ public class RunOfflineAirPollutionAnalysisByVehicleCategoryTest{
 			String runDirectory = "https://svn.vsp.tu-berlin.de/repos/public-svn/matsim/scenarios/countries/de/berlin/berlin-v5.4-0.1pct/output-berlin-v5.4-0.1pct/";
 			String runId = "berlin-v5.4-0.1pct";
 			String hbefaFileWarm = "https://svn.vsp.tu-berlin.de/repos/public-svn/3507bb3997e5657ab9da76dbedbb13c9b5991d3e/0e73947443d68f95202b71a156b337f7f71604ae/7eff8f308633df1b8ac4d06d05180dd0c5fdf577.enc";
-			String hbefaFileCold = "https://svn.vsp.tu-berlin.de/repos/public-svn/3507bb3997e5657ab9da76dbedbb13c9b5991d3e/0e73947443d68f95202b71a156b337f7f71604ae/22823adc0ee6a0e231f35ae897f7b224a86f3a7a.enc";
+			String hbefaFileCold = "https://svn.vsp.tu-berlin.de/repos/public-svn/3507bb3997e5657ab9da76dbedbb13c9b5991d3e/0e73947443d68f95202b71a156b337f7f71604ae/ColdStart_Vehcat_2020_Average_withHGVetc.csv.enc";
 			
 			RunOfflineAirPollutionAnalysisByVehicleCategory analysis = new RunOfflineAirPollutionAnalysisByVehicleCategory(runDirectory, runId, hbefaFileWarm, hbefaFileCold, utils.getOutputDirectory());
 			analysis.run();
-			
+
 		} catch ( Exception ee ) {
 			throw new RuntimeException(ee) ;
 		}

